### ISSUE TITLE
Support for non-x86_64 platforms

### ISSUE THREAD

**Issue by leftmostcat** [kornelski/xml-rs#12]
The documentation notes (via Cargo.toml) that x86_64 is the only supported target. Is this a matter of lack of testing on other platforms, or does the code itself depend on x86_64? I am interested in consuming the crate in an environment that would potentially be cross-architecture.

**Comment by kornelski** [comment]
No, this isn't meant to describe platform support. Quite the opposite — the library is identical on all platforms, so I want to save docs.rs wasted work to generate same docs many times.
